m/MATLAB/README.md

```diff
@@ -0,0 +1,43 @@
+# Installation instructions Matlab
+
+These instructions should be the same for all MATLAB versions. If future versions allow command line installations, this
+README should become obsolete.
+
+Matlab requires a manual installation with a GUI, as the command line options that Matlab provides do not seem to work.
```

Comment:
Did you try the command-line installation using EasyBuild? It seems to work for `MATLAB-2022b.eb` when I provide the appropriate values for `EB_MATLAB_KEY`, `EB_MATLAB_LICENSE_SERVER`, and `EB_MATLAB_LICENSE_SERVER_PORT`. The `support_packages/matlab_parallel_server` directory is however missing, I guess that still has to be done separately. See https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/m/MATLAB#license